packages/react-frontend/src/components/MyBetsFeed.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import MyBet from "./MyBet.js";
+
+function MyBetsFeed(props) {
+  const myBets = props.bets.map((bet, index) => {
+    return (
+      <MyBet
+        key={index}
+        decision={bet.decision}
+        amount={bet.amount}
+        question={bet.promptId.question}
+      />
+    );
+  });
```

Comment:
Wouldn't it be better for the MyBet component to take a Bet object as a prop instead of each individual bet field?
e.i.
\<MyBet bet={bet} />

Reply:
I have now implemented your requested change.